invadedhouse/rooms.py

```diff
@@ -175,10 +175,52 @@ def place(self, item: Item):
         self.contents.append(item)
 
     def loot(self) -> [Item]:
+        """
+        Loots this chest (removes all items contained inside)
+
+        :return: set of Items that were in this chest (empty set if nothing was inside)
+        """
```

Comment:
whats this

Reply:
These are just comments to describe how the loot function works